apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/index.tsx

```diff
@@ -76,11 +71,15 @@ function nextStepKey(step: StepKey): StepKey {
 const SyncOnboardingManual = () => {
   const { t } = useTranslation();
   const history = useHistory();
-  const [stepKey, setStepKey] = useState<StepKey>(StepKey.Paired);
+  const [stepKey, setStepKey] = useState<StepKey>(StepKey.SoftwareCheck);
 
   const handleSoftwareCheckComplete = useCallback(() => {
     // TODO: put this line instead
     // setStepKey(nextStepKey(StepKey.SoftwareCheck));
+    setStepKey(StepKey.Applications);
+  }, []);
+
+  const handleInstallRecommendedApplicationComplete = useCallback(() => {
     setStepKey(StepKey.Ready);
```

Comment:
Same here, with `nextStepKey` you don't need to know the next step, you just say you go next